GoogleChromeLabs/ps-analysis-tool
# Feature: Cookie table settings should be made persistent

## Description
In the application tab, the cookies table settings like column-sizing, column-sorting, frame selected etc. are persistent i.e. they are still applied once the user closes the devtools and then reopens it. We want to apply that same functionality to our tool.

## Relevant Technical Choices
- Storing the tab cookie table settings along with the cookies in the storage.

## Testing Instructions
- Open the terminal and run `npm I && npm run dev`.
- Open `aljazeera.com` and go to the Privacy Sandbox tab.
- Now open the cookies drop-down and select a frame.
- Play around in the cookie table and select different sorting methods, toggle columns etc.
- Close the DevTools.
- Now reopen the DevTools Navigate to the Privacy Sandbox tab you should see the Tab will be the same way as you left when you closed the DevTools.

## Screenshot/Screencast

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/00f48b05-6423-4b40-b55e-8b179930d2cf

---

Fixes #74
